rosdep/python.yaml

```diff
@@ -6251,6 +6260,15 @@ python3-helyos_agent_sdk-pip:
   '*':
     pip:
       packages: [helyos_agent_sdk]
+python3-hid:
```

Comment:
Can you please add the debian/fedora/rhel options as suggested by the tooling?

Reply:
I added the debian/fedora options as well.
Unfortunately, since I don't have account for rhel, I couldn't add the option for it.